pom.xml

```diff
@@ -124,24 +144,28 @@
                 </plugins>
             </build>
         </profile>
+        
     </profiles>
 
     <distributionManagement>
-        <snapshotRepository>
-            <id>ossrh</id>
-            <url>https://oss.sonatype.org/content/repositories/snapshots</url>
-        </snapshotRepository>
-        <repository>
-            <id>ossrh</id>
-            <url>https://oss.sonatype.org/service/local/staging/deploy/maven2/</url>
-        </repository>
-    </distributionManagement>
 
+			<!-- This way we can play with settings and profile and select alternative distro repos. -->
+			
+      <snapshotRepository>
+          <id>${distro.snap.id}</id>
+          <url>${distro.snap.url}</url>
+      </snapshotRepository>
+      <repository>
+          <id>${distro.id}</id>
+          <url>${distro.url}</url>
+      </repository>
+      
+    </distributionManagement>
+      
     <scm>
-        <url>https://github.com/harti2006/neo4j-server-maven-plugin</url>
+        <url>https://github.com/Rothamsted/neo4j-server-maven-plugin</url>
```

Comment:
Please revert modifications to the scm details. I guess if you want to keep those kind of changes in your fork, you have to move them to a separate "release" branch to not mix them into the features that should be merged upstream.

Reply:
For some reason the GH pull mechanism is not pulling the branch where I fixed these issues. Will check again.